docs/content/core/system-ssh-agent.md

```diff
@@ -7,28 +7,16 @@ aliases:
 
 
 The ssh-agent service allows adding multiple keys (including the ones protected with a passphrase) to a single pool of 
-ssh keys, which can be shared across multiple projects.
+SSH keys, which can be shared across multiple projects. It runs an SSH agent process whose Unix socket is mounted into
+your project's cli container by default.
 
-The default ssh keys (`~/.ssh/id_rsa`, `~/.ssh/id_dsa`, `~/.ssh/id_ecdsa`) are loaded into the agent automatically.  
-On macOS and Windows this happens when the Docksal VM is (re)started, on Linux - whenever `fin project start` is used.
+The default SSH keys (`~/.ssh/id_rsa`, `~/.ssh/id_dsa`, `~/.ssh/id_ecdsa`, `~/.ssh/id_ed25519`) are loaded into the
+agent automatically. On macOS and Windows this happens when the Docksal VM is (re)started, on Linux - whenever
```

Comment:
Add comma after “and Windows” and change dash after Linux to comma.

Reply:
I've updated this sentence